yarn.lock

```diff
@@ -1022,7 +1022,7 @@
     unique-filename "^1.1.1"
     which "^1.3.1"
 
-"@google/earthengine@^0.1.209":
+"@google/earthengine@^0.1.217":
   version "0.1.217"
```

Comment:
@ibgreen It understand this to mean that the existing `yarn.lock` was resolving to version `0.1.217`? Usually when I update dependency versions, I see large blocks in the diff when the resolved version in the `yarn.lock` changes, but this has no diff.

Reply:
Hmm, yes there is a caret here too. I am not 100% sure but you are most probably right that someone doing a fresh install, like you recently did, would get 0.217. 

However, someone who did this a long time ago, like me, would still see 0.209. as that is compatible with both package.json and yarn.lock